src/main/java/racingcar/RacingGame.java

```diff
@@ -7,27 +7,38 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Random;
+import java.util.Scanner;
 
 public class RacingGame {
+    private static final int RANDOM_BOUND = 10;
+    private static final String NUMBER_OF_CAR_MSG = "자동차 대수는 몇 대 인가요?";
+    private static final String NUMBER_OF_TRY_MSG = "시도할 회수는 몇 회 인가요?";
 
     public static void main(String[] args) {
-        int carCount = new InputView().getCount("자동차 대수는 몇 대 인가요?");
-        int moveCount = new InputView().getCount("시도할 회수는 몇 회 인가요?");
+        Scanner scanner = new Scanner(System.in);
+        InputView carInputView = new InputView(NUMBER_OF_CAR_MSG, scanner);
+        InputView tryInputView = new InputView(NUMBER_OF_TRY_MSG, scanner);
+
         ResultView resultView = new ResultView();
+        System.out.println();
+        System.out.println("실행 결과");
```

Comment:
매직 리터럴은 상수로 치환하면 좋을 것 같아요
```suggestion
        System.out.println(RESULT_MSG);
```